<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabWidget</class>
 <widget class="QWidget" name="TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="tabButtons">
     <item>
      <widget class="QPushButton" name="addRowButton">
       <property name="text">
        <string>Add row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteRowButton">
       <property name="text">
        <string>Delete selected</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
